src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -554,16 +554,16 @@ public String getAlternativeOreName() {
     }
 
     // todo clean this up
-    public String getLocalNameForItem(Material material) {
-        String specifiedUnlocalized = "item." + material.toString() + "." + this.name;
+    public String getLocalNameForItem(@Nonnull Material material) {
+        String specifiedUnlocalized = "item." + material.getUnlocalizedName() + "." + this.name;
         if (LocalizationUtils.hasKey(specifiedUnlocalized)) return LocalizationUtils.format(specifiedUnlocalized);
         String unlocalized = findUnlocalizedName(material);
```

Comment:
That check is only for lang overrides. It would otherwise delegate to the ore prefix translation in `findUnlocalizedName()`, and those do not have mod ids associated with them.